utils/run_test.sh

```diff
@@ -52,6 +52,26 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES=${DAOS_BASE}/test_results/*.xml
+    for f in $FILES
+    do
+        if [ -f "$f" ]; then
+            echo "Processing XML $f"
+
+            SUITE=`grep "testsuite name=" "$f" | \
+                   grep -Po "name=\"\K.*(?=\" time=)"`
+
+            CLASS=`grep "<testcase classname" "$f"`
+            if [ "$CLASS" == "" ]; then
+                sed -i \
+                "s/case name/case classname=\"${COMP}.${SUITE}\" name/" "$f"
+            else
+                sed -i "s/case classname=\"/case classname=\"${COMP}./" "$f"
+            fi
+        fi
+    done
+    mv ${DAOS_BASE}/test_results/*.xml ${DAOS_BASE}/test_results/xml
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]